### PR TITLE
Add extra feature detection parameters

### PR DESCRIPTION
**Pull request by mtekman**
FOR CONTRIBUTOR:
* [X] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [X] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

Extra test file added too

### CODE REVIEW

**Comment by mtekman** [comment]
do not merge yet, some other metrics need updating

**Comment by mtekman** [comment]
Please merge

File: tools/raceid/raceid_filtnormconf.xml [hunk 1]
```diff
@@ -65,22 +65,20 @@ Rscript '$__tool_directory__/scripts/$bin' '$userconf' 2> '$outlog' > /dev/null
                         <param name="logscale" type="boolean" value="false" label="Log-transform data prior to PCA or ICA" help="" />
                     </when>
                 </conditional>
-                <param name="use_log" type="boolean" checked="false" label="Output Log?" />
```
Comment (bgruening) [inline tools/raceid/raceid_filtnormconf.xml]: Any reason why you made this non-optional?

**Comment by mtekman** [comment]
The previous filter logic seemed to always evaluate to true (I played around a lot trying to get it to work, but no luck) and also it has useful metrics that would likely be wanted by default (amount filtered, percentages, etc).

**Pull request closed without merging**